devops/docker_build/dev.Dockerfile

```diff
@@ -57,6 +57,9 @@ COPY devops/docker_build/create_users.sh .
 RUN /bin/bash -c "./create_users.sh"
 COPY devops/docker_build/etc_sudoers /etc/sudoers
 
+# - Change the ownership `/etc/sudoers` to root.
```

Comment:
Let's explain why it's needed, rather than what it does.
E.g., I would imagine that it doesn't matter who has the ownership of this, but yes it's best to have it too root.
But is it a must have?

Reply:
Added the explanation of this fix.